### ISSUE TITLE
[Bug]: lag?

### ISSUE THREAD

**Issue by SmRzT** [Yimura/YimMenu#1140]
### Describe the bug

(1)When ever i play with the Yim Menu im getting severe lag, like when playing with a friend he will be walking fine for 10s and then stand still for 30s then will be where he is on his screen and for him im moving fluently. However if i unload the menu there is no lag at all

Im not sure if this is only in missions or if a certain options trigger the lag like TP or its just happening with him i ll need to play more but they aren't on atm ill see in a bit with my other friend to see if this also happens

~~(2)And the second part of the title, im not sure if you'd deal with it but when trying to load Yim menu then N3NY00 or vise-versa the game will crash~~

~~I am using latest vers of this post (Nightly) with latest Xenos injector
And using 5.8 N3NY00 if youd want that~~

### Steps To Reproduce

~~Explained in (2)~~

### Expected Behavior

No desync( if thats the right word/lag) 
~~Prob not a crash~~

### Screenshots and/or Logs

```

```
When they are on i can provide a vid (1)

~~I can upload logs of N3NY00 crash if youd like idk where to find them tho~~

### Store

Epic Games

### Additional context

My friend uses The steam ver with no mods

### Build

- [X] I've confirmed the issue exists on the latest version of YimMenu

**Comment by 1642054178** [comment]
Why don't you go ask n3ny00？

**Comment by MarkEcza** [comment]
1. Might need more testing and confirmation.
2. Why should we care? And as i know of 'n3ny00' uses the same base so it's only normal that there is going to be conflicts. You should rather complain to the 'n3ny00' developer instead of here.

**Comment by Yimura** [comment]
Nenyoo and YimMenu are not compatible as Nenyoo uses the same code base as YimMenu.

Just play with either YimMenu or Nenyoo.
In regards to the lag, I don't have any so you'll have to specify a bit more by posting your system specs.

CPU cores?
CPU threads?
Operating System?
GPU?

**Comment by SmRzT** [comment]
> Nenyoo and YimMenu are not compatible as Nenyoo uses the same code base as YimMenu.
> 
> Just play with either YimMenu or Nenyoo. In regards to the lag, I don't have any so you'll have to specify a bit more by posting your system specs.
> 
> CPU cores? CPU threads? Operating System? GPU?

I've asked Bugga and they will look into the crashing with Yim, Something with N3NY000 Coloading idk.

In regards to the lag its quite a confusing one.
the stuff i described in the OP did happen, maybe when i unloaded menu something synced me with him idktbh
From what i tested last night while playing I was walking around T-Posing so ima just assume he has some pretty dogshit wifi,I asked the other friend if he also saw this and he said "naw" but doesn't really explain the first one when unloading the menu and loading it back up 15mins later and happening 5s later

with that being said most of it is covered and with this prob being his "dogshit wifi" ima close this and if anyone else to have something like this can reopen it ig

anyways specs are:
AMD Ryzen 5 5600X ,6C,12T
AMD RX6600
Windows 11

Friend:
2700x, 8C 16T
2070 Super 
Windows 10

**Comment by SmRzT** [comment]
Lag1:
https://youtu.be/mE9F6FddZVs

Lag2, 7 mins later,1m after re-opening Yim:
(The cops are a better example in this one than the players I focused on in the map)
https://youtu.be/xBz-25BKBWw

Ig ill leave this with your thoughts and ima get back on with my friend who saw me T-posing and see if its his shitty wifi or im being desynced with menu open

I use kiddions on the side but nothing really open/in use on it, with 2 scripts not actively doing anything (Storm,Ultimate Menu)

**Comment by Yimura** [comment]
Not sure what could be causing these issues, it's probably just a bad host not syncing frequently enough.

I personally have never experienced lag this bad only when playing in regions where the host wouldn't be anywhere near my real location.

**Comment by SmRzT** [comment]
> Not sure what could be causing these issues, it's probably just a bad host not syncing frequently enough.
> 
> I personally have never experienced lag this bad only when playing in regions where the host wouldn't be anywhere near my real location.

Yea its kinda a weird one, maybe only in missions? I noticed this also in the last dose setups with KGBDuMbZ, Same thing When unloading menu everything will sync back up.

The first 2 clips I believe the Host Weidevieh Is from Germany, I am from UK 

And in clip 3 below My friend KGBDuMbZ (who notably has shite wifi like 50mbps ,which I have described in prev post) is the host Also from UK But in this post Other people are moving Except him and the NPC's and when the teamate dies my lives don't change until the menu is unloaded and KGBDuMbZ tp to his pos and npc are able to be killed.

I believe in Lag1 Weidevieh isn't moving and maybe not the other two, kinda hard to tell. Until menu is unloaded

But in Lag2 Weidevieh ,Same Session and host, is moving but the other two aren't neither the NPC's.(although his character could just be falling), Until menu is unloaded


Lag3:
https://youtu.be/y_Xc6x-3jOE
